### PR TITLE
S05US22

### PR DESCRIPTION
**Pull request by Chikoulat**
FEAT: Added CSS and upload for candidate and company inscription.FEAT: Added CSS and upload for candidate and company inscription.

### CODE REVIEW

**Review by ioayoub — APPROVED** [review]

**Review by StalkerVince — APPROVED** [review]

File: backend/src/routers/jobRouter.js [hunk 1]
```diff
@@ -6,7 +6,6 @@ const {
   browse,
   browseFilters,
   read,
-  readOffer,
```
Comment (YOUNS28100) [inline backend/src/routers/jobRouter.js]: add please

File: backend/src/routers/jobRouter.js [hunk 2]
```diff
@@ -19,8 +18,7 @@ const { verifyToken } = require("../middlewares/hashPassword");
 router.get("/", browse);
 router.get("/count", browseCount);
 router.get("/searchPage", browseFilters);
-router.get("/companyoffers", read);
-router.get("/:id", readOffer);
```
Comment (YOUNS28100) [inline backend/src/routers/jobRouter.js:23]: add please

File: frontend/src/components/company/CandidatList.jsx [hunk 3]
```diff
@@ -6,6 +6,7 @@ import style from "../../assets/styles/messagePage.module.scss";
 function CandidatList() {
   const { auth } = useOutletContext();
   const [candidates, setCandidates] = useState();
+  const candidate = [candidates];
```
Comment (YOUNS28100) [inline frontend/src/components/company/CandidatList.jsx:9]: no need for new const here, better to rename the state

Comment (Chikoulat) [reply frontend/src/components/company/CandidatList.jsx:9]: i need it to create an array of objects as the state only recovers objects

**Review by YOUNS28100 — APPROVED** [review]

**Pull request merged**